common/components/multi-file-upload/multi-file-upload.js

```diff
@@ -315,7 +315,7 @@ MultiFileUpload.prototype = {
     if (errorData.length === 0) {
       errorData.push({
         text: 'Something went wrong',
-        href: '#document_upload',
+        href: '#' + this.$fileInput.id,
```

Comment:
Extremely minor, but maybe let's use a string template? `#${this.$fileInput.id}` for consistency's sake? If you don't want to do it no problem, string concatenation is arguably more readable.